sparse_strips/vello_common/src/pico_svg.rs

```diff
@@ -9,7 +9,13 @@
 //! for demonstration purposes. It supports basic SVG features like paths,
 //! fill, stroke, and grouping.
 
-use std::str::FromStr;
+extern crate std;
+
+use alloc::boxed::Box;
+use alloc::vec;
+use alloc::vec::Vec;
+use core::str::FromStr;
+use std::eprintln;
```

Comment:
:laughing:

Reply:
This makes it clear now what the offending code is. :)